index.html

```diff
@@ -72,11 +72,9 @@
   <body>
     <div class="dropdown hide">
       <ul class="dropdown-list">
-        <li class="dropdown-item">
-          <a class="dropdown-link" href="https://my.realdevsquad.com/">
-            My Profile
-          </a>
-        </li>
+        <a class="dropdown-link" href="https://my.realdevsquad.com/">
```

Comment:
I think we can do this with the proper hierarchical way like ul > li > a , which also follows standards of HTML and will be good approach. Although we can do it like this but better approach will be good.

Reply:
Corrected as per requirements

Reply:
Satyam and Vaibhav can't we give it an approach like a `Sign Out` button?

Reply:
> 

Sunny your approach will not be good for accessibility I think because you are using li as anchor tag. So the screen readers and crawler will not be able to recognize it as a anchor rather it will be considered a li element.

Reply:
You are right, but the SIGNOUT button remains the same. So change the approach of the SIGNOUT button as well.

Reply:
Actually My Profile button can be changed to event handler like the SIGNOUT button. Also for SIGNOUT button we are making an api call first that's why anchor tag will have a problem, as it is not redirecting rather waiting for the api request to resolve and then reloading.

Reply:
It's not a `button` it is `li` right?

Reply:
Yes it's li elemenwnt but has an onclick event handler

Reply:
sort of working like a button